apps/proto/lib/lexical/proto/alias.ex

```diff
@@ -1,12 +1,15 @@
 defmodule Lexical.Proto.Alias do
   alias Lexical.Proto.CompileMetadata
   alias Lexical.Proto.Field
+  alias Lexical.Proto.Macros.Typespec
 
   defmacro defalias(alias_definition) do
     caller_module = __CALLER__.module
     CompileMetadata.add_type_alias_module(caller_module)
 
     quote location: :keep do
+      unquote(Typespec.t(alias_definition, __CALLER__))
```

Comment:
I found `Typespec.t` a little confusing at first. What if we just call `typespec` directly?

```elixir
@type t :: unquote(Typespec.typespec(alias_definition, __CALLER__))
```